vm/src/stdlib/collections.rs

```diff
@@ -447,7 +493,8 @@ mod _collections {
     struct PyDequeIterator {
         position: AtomicCell<usize>,
         status: AtomicCell<IterStatus>,
-        length: usize, // To track length immutability.
+        length: usize,
+        // To track length immutability.
```

Comment:
this comment looks to be placed wrong

Reply:
I fixed this, and also place new comment about state using this placement. Thanks!